Languages/po/de.po

```diff
@@ -202,6 +206,8 @@ msgid ""
 "%s is not a directory, failed to move to *.original.\n"
 " Verify your write permissions or move the file outside of Dolphin"
 msgstr ""
+"%s ist kein Verzeichnis. *.original konnte nicht verschoben werden.\n"
+" Überprüfen deine Schreibrechte oder verschieben Sie die Datei in einen Verzeichnis außerhalb von Dolphin"
```

Comment:
" Überprüfen Sie Ihre Schreibrechte oder verschieben Sie die Datei in ein Verzeichnis außerhalb von Dolphin"